functions/src/api/v1/group/getGroup.ts

```diff
@@ -0,0 +1,65 @@
+import { wrap } from 'async-middleware'
+import { Request, Response } from 'express'
+import { UID } from '../../../types/uid'
+import { GroupId } from '../../../types/Group'
+import { assertGroupReadAllowed } from './groupRights'
+import { GroupDao } from '../../../db/GroupDao'
+
+/**
+ * @swagger
+ * /v1/groups/{groupId}/:
+ *   get:
+ *     description: Get the given group. Members user ids will be returned for regular members and all members details will be returned for the owner.
```

Comment:
shouldn't only the `owner` be able to see the members list ?

Reply:
it is, wrong desc ty